package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "@curriculumassociates/createjs-accessibility",
-  "version": "1.0.6",
+  "version": "1.0.7",
```

Comment:
Has any of the dist code for [https://github.com/CurriculumAssociates/createjs-accessibility changed]()? If not, I don't think we need to bump the version in the package.json (since there's no difference in the distributed package/module).

Reply:
ah gotcha, reverted back to 1.0.6!